java/src/jmri/jmrit/roster/PrintRosterEntry.java

```diff
@@ -59,9 +68,10 @@ public PrintRosterEntry(RosterEntry rosterEntry, JmriJFrame parent, String filen
                 return;
             }
             if ((base = root.getChild("programmer")) == null) {
-                log.error("xml file top element is not programmer");
+                log.error("xml file top element is not a programmer");
```

Comment:
In this case we are looking for an element named "programmer" (wrapping it in quotes would be more clarifying than adding the indefinite article).